Test.Utility.Test/Stubs/ITestInterface.cs

```diff
@@ -0,0 +1,10 @@
+namespace Messerli.Test.Utility.Test.Stubs
+{
+    public interface ITestInterface
+    {
+    }
+
+    public interface ITestInterface2
+    {
+    }
+}
```

Comment:
Add terminating newline.
There seems to be no .editorconfig

Reply:
Addressed in https://github.com/messerli-informatik-ag/test-utility/pull/24/commits/c8be4571c294ab68dabf6b628a4ae6b21c0d5eda